From f2e3cd61dea9b94f7f4634930fe57316d9f5ee4d Mon Sep 17 00:00:00 2001
From: Hein-Pieter van Braam <hp@tmm.cx>
Date: Wed, 10 May 2017 01:27:37 +0200
Subject: [PATCH libwacom 3/7] Add isdv4-4814 tablet description

This device was found on a HP Elite x2 1012 G1. It is a copy of the
previously existing 4800 definition.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit a5ec9b783c7c5ea16c90bad82bf5a2b6eb71b57c)
---
 data/isdv4-4814.tablet | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 data/isdv4-4814.tablet

diff --git a/data/isdv4-4814.tablet b/data/isdv4-4814.tablet
new file mode 100644
index 0000000..ad6d0fa
--- /dev/null
+++ b/data/isdv4-4814.tablet
@@ -0,0 +1,14 @@
+# Active electrostatic (AES) sensor used by the HP Elite x2 1012 G1
+
+[Device]
+Name=Wacom ISDv4 4814
+DeviceMatch=i2c:056a:4814
+Class=ISDV4
+Width=10
+Height=7
+IntegratedIn=Display;System
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
-- 
2.13.5

